x-pack/plugins/reporting/server/export_types/common/generate_png.ts

```diff
@@ -15,6 +15,8 @@ import { LayoutParams, LayoutSelectorDictionary, PreserveLayout } from '../../li
 import { getScreenshots$, ScreenshotResults } from '../../lib/screenshots';
 import { ConditionalHeaders } from '../common';
 
+export const REPORTING_TRANSACTION_TYPE = 'reporting';
```

Comment:
I think we have a more suitable place to add this constant in `x-pack/plugins/reporting/common/constants.ts`.

It could be added to the file as:
```diff
 export const PLUGIN_ID = 'reporting';
 
+export const REPORTING_TRANSACTION_TYPE = PLUGIN_ID;
+
```